README.md

```diff
@@ -1,9 +1,63 @@
 # Tail
 
-A very simple app to display logs generated by [Prow](https://github.com/kubernetes/test-infra/tree/master/prow)
+A very simple app to display logs generated by [Prow](https://git.k8s.io/test-infra/prow).
+
+## Why use Tail
+
+By default, logs generated by Prow are stored in a GCS bucket and are displayed
+publicly by a tool called [Gubernator](https://git.k8s.io/test-infra/gubernator).
+It is not possible to display logs privately using it.
+
+Tail solves this by:
+- Providing GitHub-based user authorization and restricting access to logs from
+private repos to only members of a particular organisation.
+- Publicly displaying logs from public repos, without any need for
+authorization in these cases.
+
+## Installation
+
+Tail requires Go version 1.7+.
```

Comment:
How do you know that? :D

Reply:
> How do you know that?

Because we use context and that's only present in 1.7+ :)